scripts/track.py

```diff
@@ -380,19 +385,28 @@ def _grow_step(self, sprouts, states, step_size):
     def grow(self, step_size):
         self.sprouts, self.states = self._grow_step(self.sprouts, self.states, step_size)
 
+    def _keep(self, idx):
+        # Update remaining sprouts and their states.
+        self.sprouts = self.sprouts[idx]
+        self._states = [s[idx] for s in self._states]
+
+        # Rewrite history
+        for i, state in enumerate(self._history):
+            self._history[i] = [s[idx] for s in state]
+
     def harvest(self):
         undone, done, stopping_flags = self.is_stopping(self.sprouts)
 
-        tractogram = Tractogram(streamlines=compress_streamlines(list(self.sprouts[done])),
-                                data_per_streamline={"stopping_flags": stopping_flags})
+        # TODO: Should we keep last point since it almost surely raised the stopping flag?
```

Comment:
@ppoulin91 keep this in mind, but it should influence the Tractometer scores that much.